client/index.go

```diff
@@ -40,6 +40,15 @@ type IndexDescription struct {
 	Unique bool
 }
 
+// PolicyDescription describes an index.
+type PolicyDescription struct {
+	// ID is the policyID that is unique to every policy.
+	ID string
```

Comment:
suggestion: I guess this ID is global, but would be nice to the documentation explicitly mention it

Reply:
Is it global? It might seem that way, and actually might end up being global. But it's not discussed or explicitly planned out so I would like to avoid documenting something that isn't for sure.

Leaving as is unless others have a preference for it.